src/bn_accounts.erl

```diff
@@ -14,8 +14,24 @@
 %%
 
 handle_rpc(<<"account_get">>, {Param}) ->
+    Chain = blockchain_worker:blockchain(),
+    case ?jsonrpc_get_param(<<"height">>, Param, false) of
+        false ->
+            Ledger = blockchain:ledger(Chain),
+            Height = bn_txns:follower_height();
+        HeightParam ->
+            Height = HeightParam,
```

Comment:
I'd suggest using

``Height = case ... ` to get the variable assigned just once instead of the dance inside each match arm